examples/webgl_effects_ascii.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - effects - ascii</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				padding:0;
+				margin:0;
+				font-weight: bold;
+				overflow:hidden;
+			}
+
+			#info {
+				position: absolute;
+				top: 0px; width: 100%;
+				color: #000000;
+				padding: 5px;
+				font-family:Monospace;
+				font-size:13px;
+				text-align:center;
+				z-index:1000;
+			}
+
+			a {
+				color: #ff0000;
+			}
+
+			#webglmessage a { color:#da0 }
+		</style>
+	</head>
+
+	<body>
+		<div id="info"><a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - effects - ascii</div>
+		<script src="../build/three.js"></script>
+
+		<script src="js/controls/TrackballControls.js"></script>
+		<script src="js/effects/AsciiEffect.js"></script>
+
+		<script src="js/WebGL.js"></script>
+
+		<script>
+
+			if ( WEBGL.isWebGLAvailable() === false ) {
+
+				document.body.appendChild( WEBGL.getWebGLErrorMessage() );
+
+			}
+
+			var camera, controls, scene, renderer, effect;
+
+			var sphere, plane;
+
+			var start = Date.now();
+
+			init();
+			animate();
+
+			function init() {
+
+				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 1, 1000 );
+				camera.position.y = 150;
+				camera.position.z = 500;
+
+				controls = new THREE.TrackballControls( camera );
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0xf0f0f0 );
+
+				var light = new THREE.PointLight( 0xffffff );
+				light.position.set( 500, 500, 500 );
+				scene.add( light );
+
+				var light = new THREE.PointLight( 0xffffff, 0.25 );
+				light.position.set( - 500, - 500, - 500 );
+				scene.add( light );
+
+				sphere = new THREE.Mesh( new THREE.SphereBufferGeometry( 200, 20, 10 ), new THREE.MeshLambertMaterial() );
+				scene.add( sphere );
+
+				// Plane
+
+				plane = new THREE.Mesh( new THREE.PlaneBufferGeometry( 400, 400 ), new THREE.MeshBasicMaterial( { color: 0xe0e0e0 } ) );
+				plane.position.y = - 200;
+				plane.rotation.x = - Math.PI / 2;
+				scene.add( plane );
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				// document.body.appendChild( renderer.domElement );
+
+				effect = new THREE.AsciiEffect( renderer );
+				effect.setSize( window.innerWidth, window.innerHeight );
+				document.body.appendChild( effect.domElement );
```

Comment:
>appendChild( effect.domElement );

Can the code be modified so you can add `renderer.domElement `, instead?

Reply:
I don't think that's possible. `AsciiEffect` creates a custom `domElement` (a div container) where the ASCII elements are placed.

Reply:
Hmmm. Users copy code. How about  inserting this comment:
```javascript
// Special case: append effect.domElement, instead of renderer.domElement. AsciiEffect creates a custom domElement (a div container) where the ASCII elements are placed.
```

Reply:
Good idea! :+1: